msgraph/authentication_strength_policy.go

```diff
@@ -141,6 +141,38 @@ func (c *AuthenticationStrengthPoliciesClient) Update(ctx context.Context, Authe
 	return status, nil
 }
 
+// Update amends an existing AuthenticationStrengthPolicy's allowed combinations
+func (c *AuthenticationStrengthPoliciesClient) UpdateAllowedCombinations(ctx context.Context, Policy AuthenticationStrengthPolicy) (int, error) {
```

Comment:
Can we camelcase this local variable?

```suggestion
func (c *AuthenticationStrengthPoliciesClient) UpdateAllowedCombinations(ctx context.Context, policy AuthenticationStrengthPolicy) (int, error) {
```